code/mobile/android/PhoneVR/app/src/main/res/values/strings.xml

```diff
@@ -1,5 +1,6 @@
 <resources>
-    <string name="app_name">PhoneVR</string>
+    <string name="app_name_full">PhoneVR</string>
+    <string name="app_name_lite">PhoneVR Lite</string>
```

Comment:
PhoneVR Lite seems misleading, i think PhoneVR noGvr would be much more apt

Reply:
Done